endpoints/info/bidders_test.go

```diff
@@ -12,238 +12,470 @@ import (
-			givenAliases: map[string]string{"b": "a"},
-			expected:     `["a","b"]`,
+			name:         "alias-mixed",
+			givenBidders: config.BidderInfos{"a": disabledCore, "b": disabledAlias, "c": enabledCore, "d": enabledAlias},
+			expected:     `["a","c"]`,
+		},
+	}
+
+	for _, test := range testCases {
+		t.Run(test.name, func(t *testing.T) {
+			result, err := prepareBiddersResponseAllBaseOnly(test.givenBidders)
+			assert.NoError(t, err)
+			assert.Equal(t, []byte(test.expected), result)
+		})
+	}
+}
+
+func TestPrepareBiddersResponseEnabledOnly(t *testing.T) {
+	var (
+		enabledCore   = config.BidderInfo{Disabled: false}
+		enabledAlias  = config.BidderInfo{Disabled: false, AliasOf: "something"}
+		disabledCore  = config.BidderInfo{Disabled: true}
+		disabledAlias = config.BidderInfo{Disabled: true, AliasOf: "something"}
+	)
+
+	testCases := []struct {
+		name                string
+		givenBidders        config.BidderInfos
+		givenRequestAliases map[string]string
+		expected            string
+	}{
+		{
+			name:                "none",
+			givenBidders:        config.BidderInfos{},
+			givenRequestAliases: nil,
+			expected:            `[]`,
 		},
 		{
-			description:  "With Aliases - Many",
-			givenBidders: config.BidderInfos{"a": enabled, "b": disabled},
-			givenAliases: map[string]string{"x": "a", "y": "b"},
-			expected:     `["a","b","x","y"]`,
+			name:                "core-one-enabled",
+			givenBidders:        config.BidderInfos{"a": enabledCore},
+			givenRequestAliases: nil,
+			expected:            `["a"]`,
 		},
 		{
-			description:  "With Aliases - Sorted",
-			givenBidders: config.BidderInfos{"z": enabled},
-			givenAliases: map[string]string{"a": "z"},
-			expected:     `["a","z"]`,
+			name:                "core-one-disabled",
+			givenBidders:        config.BidderInfos{"a": disabledCore},
+			givenRequestAliases: nil,
+			expected:            `[]`,
+		},
+		{
+			name:                "core-one-mixed",
+			givenBidders:        config.BidderInfos{"a": disabledCore, "b": enabledCore},
+			givenRequestAliases: nil,
+			expected:            `["b"]`,
+		},
+		{
+			name:                "core-one-mixed-sorted",
+			givenBidders:        config.BidderInfos{"z": enabledCore, "a": enabledCore},
+			givenRequestAliases: nil,
+			expected:            `["a","z"]`,
+		},
+		{
+			name:                "alias-one",
+			givenBidders:        config.BidderInfos{"a": enabledAlias},
+			givenRequestAliases: nil,
+			expected:            `["a"]`,
+		},
+		{
+			name:                "alias-mixed",
+			givenBidders:        config.BidderInfos{"a": disabledCore, "b": disabledAlias, "c": enabledCore, "d": enabledAlias},
+			givenRequestAliases: nil,
+			expected:            `["c","d"]`,
+		},
+		{
+			name:                "alias-mixed-sorted",
+			givenBidders:        config.BidderInfos{"z": enabledAlias, "a": enabledCore},
+			givenRequestAliases: nil,
+			expected:            `["a","z"]`,
+		},
+		{
+			name:                "defaultequest-one",
```

Comment:
Should this and the below be `defaultrequest` rather than `defaultequest`?

Reply:
Yes. Yes it should. Nice catch. Fixed.